src/test/java/com/cdancy/bitbucket/rest/TestUtilities.java

```diff
@@ -303,6 +303,28 @@ public static synchronized void terminateGeneratedTestContents(final BitbucketAp
             assertThat(deleteStatus.value()).isTrue();
             assertThat(deleteStatus.errors()).isEmpty();
         }
+
+        // delete the local git directory
+        assertThat(deleteDirectory(gitDirectory)).isTrue();
+    }
+
+    /**
+     * Delete a directory tree.
+     *
+     * @return True when the path is deleted, false otherwise.
+     */
+    public static boolean deleteDirectory(File path) {
+        if (path.exists()) {
+            File[] files = path.listFiles();
+            for (int i = 0; i < files.length; i++) {
+                if (files[i].isDirectory()) {
+                    deleteDirectory(files[i]);
+                } else {
+                    files[i].delete();
+                }
+            }
+        }
+        return (path.delete());
```

Comment:
This is good but if you wanted to add apache-io lib to the test dependencies I wouldn't mind that and you could this with a single call like: `Files.deleteDirectory(some-dir)`. I'm fine with this though but it would be one less thing we have to maintain.

Reply:
Sure thing.